### DISCUSSION TITLE
IP Address range for Micro

### DISCUSSION THREAD

**Discussion by Prabhakar-mishra360** [deta/cloud-docs#429] [Help]
Hi Team,
My micro making a remote MySql connection and since I need to whitelist the IP on MySql server and Micro IP changing with every deta deploy may I know the range in which all micro placed so I can allow that range from my mysql database.

Thanks

**Comment by abdelhai** [comment]
hi, mysql is not supported on deta as per docs. micros do now have a static ip range unfortunately.

**Reply by BrahianVT** [reply]
Hi  By default, there is no a specific range of public IPs. Instead, they can use an IP from a subnet of a public cloud environment.
Check with the deta.sh team  I guess @abdelhai  on how to know the the list of public IPs they use for the outbound requests. They might offer a service for that (Heroku does) or suggest some SOCK-5 proxy that will route all the deta.sh request through a predefined list of proxy IPs.
